### DISCUSSION TITLE
[预览] v2.7.0

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#4096] [分享 / Share]
包含 v2.7.0 的所有更新内容.

- 新增功能 `组件二等分`, 在出现问题时可帮助定位是哪个组件导致的问题. (#3829, PR #3965 by [JLoeve](https://github.com/LonelySteve))
- 优化了 `loadFeatureCode` 的代码. (PR #4013 by [timongh](https://github.com/timongh))
- 搜索栏在搜索 ID 时的优化, 新增功能可以通过安装对应的插件来使用: (#677)
  - 搜索 av / BV 号时, 支持显示视频标题
  - 支持搜索专栏 cv 号 / 文集 rl 号
  - 支持搜索音频 au 号 / 播放列表 am 号
  - 支持搜索番剧 ss / ep 号, 或番剧详情 md 号
- `自定义顶栏` 的收藏和稍后再看支持显示数量. (#4069)
- 优化了 `网址参数清理` 的逻辑, 通过 Hook History API 来减少出现重复的历史记录. (#4039)
- 新增功能 `隐藏热搜`. (#3744)
> 隐藏搜索栏和搜索页面中的 `bilibili 热搜`.
- 新增功能 `隐藏视频分享`. (#3663)
> 隐藏视频和番剧播放器下方的分享按钮.
- `自定义顶栏` 的订阅支持过滤 在看 / 看过 / 想看. (#3217)
- 新增插件 `快捷键扩展 - 开关灯`. (#3587)
> 在快捷键的动作列表里添加一个 "开关灯".

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.7.0-preview'>[预览] v2.7.0</a>.</em>

**Comment by kethoven** [comment]
什么时候，能够将新版的list播放页面的“外置稍后再看”“快速收藏”匹配。
还有就是“稿件投诉”能够隐藏掉吗？

**Reply by Geeyun-JY3** [reply]
这不是搞了个看板么：https://github.com/users/the1812/projects/1/views/3?pane=issue&itemId=23893532

另外有需求建议我还是建议另开 discussion。

**Comment by Geeyun-JY3** [comment]
am 号的相关描述还是叫「歌单」吧（毕竟官方提到这东西都是这么叫的），而且感觉「播放列表」指代太模糊了。
![image](https://user-images.githubusercontent.com/49396421/228880840-37fd89de-d4a0-4e84-8186-3ff8ce6d5fb1.png)
![image](https://user-images.githubusercontent.com/49396421/228879612-cf7a740f-55d1-4d22-87d9-495ffe70fe1d.png)

**Comment by pbxs666** [comment]
**隐藏视频分享**在旧版播放页面失效；
**隐藏热搜**在旧版搜索页面失效。

**Comment by Geeyun-JY3** [comment]
如果 ss / ep / md 号搜索跳转无法区分是番剧、电影、电视剧、纪录片、综艺，相关文案只提「番剧」略怪，要不都提一下？
